lib/web_ui/lib/src/engine/window.dart

```diff
@@ -13,16 +13,19 @@ const bool _debugPrintPlatformMessages = false;
 /// This may be overridden in tests, for example, to pump fake frames.
 ui.VoidCallback? scheduleFrameCallback;
 
+typedef _UrlStrategyListener = void Function(JsUrlStrategy);
+
+// KEEP THIS JS NAME IN SYNC WITH flutter_web_plugins.
```

Comment:
If we have a test for this on the framework side we may not need the all caps comment :)

Reply:
Let's not use screaming caps. Normal style is OK here.